spec/links_spec.rb

```diff
@@ -6,14 +6,14 @@
   let(:readme) { File.open("README.md").read }
   let(:list) { AwesomeListRender.parse(readme) }
 
-  it 'must not appears twice' do
-    list_links = list.project_links.map { |l| l['href'] }
+  it 'must not appear twice' do
+    list_links = list.project_links.map { |l| l.map{|ll| ll['href'] }}.flatten
```

Comment:
Both the variable names `l` and `ll` are somewhat confusing. To extract all the links should we move it to a helper function?

Since you also use the same in line 16, maybe moving it to a helper function is a good idea.